src/components/right-panel-editors/media-library.tsx

```diff
@@ -176,36 +183,37 @@ const MediaLibrary: React.FC<Props> = ({
    */
   const closeUploadDialog = () => {
     setUploadDialogOpen(false);
-  }
+  };
 
   /**
    * Save event handler for upload dialog
-   * 
+   *
    * @param files uploaded files
    * @param _key key (unused)
    */
   const onUploadSave = async (files: File[], _key?: string) => {
-    const parentFolder = folder
-      .replace(/\/{1,}/g, "/")
-      .replace(/^\//, "")
-      .replace(/\/$/, "");
-
-    await Promise.all(files.map(async (file) => {
-      const presignedPostResponse = await FileUpload.getPresignedPostData(parentFolder, file);
-      if (presignedPostResponse.error) {
-        throw new Error(presignedPostResponse.message || "Error when creating presigned post request");
-      }
+    const parentFolder = folder.replace(/\/{1,}/g, "/").replace(/^\//, "").replace(/\/$/, "");
+
+    await Promise.all(
+      files.map(async (file) => {
+        const presignedPostResponse = await FileUpload.getPresignedPostData(parentFolder, file);
+        if (presignedPostResponse.error) {
+          throw new Error(
+            presignedPostResponse.message || "Error when creating presigned post request"
```

Comment:
Should this error be localized?

Reply:
Probably yes, but it does go out of the scope of this.